application/views/admin/expressions/test/survey_logic_file.php

```diff
@@ -32,4 +32,27 @@
             </div>
         </div>
     </div>
-</div>
\ No newline at end of file
+</div>
+
+<style>
```

Comment:
> BREAKING BAD NEWS for "style in body" lovers: W3C has recently [lost the HTML war](https://techxplore.com/news/2019-06-w3c-whatwg-agreement-version-html.html) against WHATWG, whose versionless HTML "Living Standard" [has now become](https://www.w3.org/2019/04/WHATWG-W3C-MOU.html) the official one, which, alas, [does not allow](https://html.spec.whatwg.org/#the-style-element) STYLE in the BODY. The short-lived happy days are over. ;) The [W3C validator](https://validator.w3.org/nu/about.html) also works by the WHATWG specs now. (Thanks @FrankConijn for the heads-up!)

https://stackoverflow.com/questions/2830296/using-style-tags-in-the-body-with-other-html

Reply:
We follow WHATWG now ? God news, really good news… … …

Reply:
Created fix: https://github.com/LimeSurvey/LimeSurvey/pull/2761